### ISSUE TITLE
Deployment of cosmosage on Jetstream

### ISSUE THREAD

**Issue by zonca** [tijmen/cosmosage#2]
@tijmen I have invited you to the allocation on Jetstream.
You should be able to login to Exosphere:
https://jetstream2.exosphere.app/exosphere/home

and spin up a GPU instance, start small for testing purposes then we can scale it up.

**Comment by zonca** [comment]
@tijmen I see that openwebui is running fine. Is Jetstream working properly?

**Comment by tijmen** [comment]
Yes, it's been working well. I just upgraded to the latest open-webui yesterday.
